backend/local/backend_apply_test.go

```diff
@@ -33,7 +35,7 @@ func TestLocal_applyBasic(t *testing.T) {
 		"ami": cty.StringVal("bar"),
 	})}
 
-	op, configCleanup := testOperationApply(t, "./testdata/apply")
+	op, configCleanup, done := testOperationApply(t, "./testdata/apply")
```

Comment:
This is purely a matter of curiosity: Would neglecting to call the `done` func cause any issues in these tests? I see a mix of `defer done()` and  `done(t).Stderr()...` depending on the situation, and I'm worried for future-me making changes and forgetting.

Reply:
Yes, it will cause a resource leak. Because the clever streams testing code opens pipes and consumes data into a buffer in a goroutine, if we don't call `done(t)` _somewhere_ I think those pipes and buffers will remain allocated until the tests complete.

That said, I am pretty sure I have failed to call `done(t)` in a few other spots, and nothing has exploded yet. Those ought to be fixed when the equivalent UI test code is replaced, as we put views in place for the other commands.

Reply:
That makes sense. I'd rather this be a single-use-pattern function; ie we _have _ to call `defer done()` and have something else to pull the output from, because i can see myself messing this up in the future. But it's fine as-is, just something to keep an eye out for!